public/app/features/alerting/unified/Analytics.ts

```diff
@@ -45,9 +45,9 @@ export function withPerformanceLogging<TFunc extends (...args: any[]) => Promise
   };
 }
 
-export async function isNewUser(userId: number) {
+export async function isNewUser() {
   try {
-    const { createdAt } = await getBackendSrv().get(`/api/users/${userId}`);
+    const { createdAt } = await getBackendSrv().get(`/api/user`);
```

Comment:
We could use RTK Query to have the caching for free

Reply:
The problem is I can't call a hook (which is what RTK Query provides) within this file as it's not a component, it's a file that exports different tracking functions to be called from any place. Seems it [can be done](https://redux-toolkit.js.org/rtk-query/usage/usage-without-react-hooks) but it looks not as straightforward, so I'd rather investigate this in a different PR.